README.md

```diff
@@ -1 +1,172 @@
-# Module-1-Capstone
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [🚀 Loom Video](#loom-video)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Module 1 Capstone Project] <a name="about-project"></a>
+
+>
+
+**[Capstone Project]** is a project made at the end of the first module at Microverse. It demonstrates what I learned in the first module.
+
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom Video <a name="loom-video"></a>
+
+> Link to the loom video of the project description.
+
+- [Loom Video Link](https://www.loom.com/share/77fc4199da5e46078766beec5377f7fc)
```

Comment:
- [ ] @Yidnekachew-cmd great work implementing the changes requested however, I found out that your presentation doesn't have a section for a piece of code you found interesting or proud of making also I couldn't see your face during the presentation which is a requirement 
![image](https://user-images.githubusercontent.com/61408860/217784063-704e7374-3562-4073-8bdc-4bccd20c1056.png)